certified-connectors/Planner/apiDefinition.swagger.json

```diff
@@ -653,6 +653,83 @@
           "type": "string",
           "x-ms-summary": "Id",
           "x-ms-visibility": "important"
+        },
+        "references": {
+          "description": "The collection of references on the task",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "resourceLink": {
+                "format": "uri",
+                "description": "A valid URL based on the HTTP/HTTPS protocols.",
+                "type": "string",
+                "x-ms-summary": "Resource link"
+              },
+              "value" : {
+                "type": "object",
+                "properties": {
+                  "alias": {
+                    "description": "A name alias to describe the reference.",
+                    "type": "string",
+                    "x-ms-summary": "Alias"
+                  },
+                  "type": {
+                    "description": "Used to describe the type of the reference.",
+                    "enum": [
+                      "PowerPoint",
+                      "Word",
+                      "Excel",
+                      "OneNote",
+                      "Project",
+                      "Visio",
+                      "Pdf",
+                      "Other"
+                    ],
+                    "x-ms-summary": "Type of the reference"
+                  }
+                }
```

Comment:
It doesn't contain type.. Shouldn't it be 
"type": {
                    "description": "Used to describe the type of the reference.",
                    "type": "string",
                    "enum": [
                      "PowerPoint",
                      "Word",
                      "Excel",
                      "OneNote",
                      "Project",
                      "Visio",
                      "Pdf",
                      "Other"
                    ],
                    "x-ms-summary": "Type of the reference"
                  }